changes/ee/feat-10408.en.md

```diff
@@ -0,0 +1 @@
+The rule engine SQL-like language has got three more built-in functions for creating values of the MongoDB date type. These functions are only useful for rules with at least one MongoDB bridge action.
```

Comment:
I have the impression that those functions when there are _only_ mongo bridge actions (not tested). (just to adjust the message here, not that there is much to do otherwise)

If the same rule using these functions has another bridge as an action, I wonder how the tuples are treated by those.  :thinking:

Reply:
I think that we should only support the tuples from `mongo_date` in mongo-related actions, indeed.  Hopefully the function name hints the user to avoid using it in other actions.  :joy_cat: